lib/ifupdown.sh.in

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 
+# This script should be symlinked into /etc/network/ip-pre-up.d/ and
```

Comment:
regarding this commit ; i've been undecided ; 
i was also trying to find an automatic formatting tool for shell scripts ; 
and trying to find a style-book [or reference] for shells ; did not find anything

i only found https://www.shellcheck.net/ ;  which checks for potential issues with scripts [ but does not do any formatting ]

so, regarding this change, I'll say ack ;
and we can make this the preferred style for shell scripts for next commits [ that touch on shell scripts ]

then later, I'll probably add stylecheck to do a check on scripts ;